internal/v1/api.yaml

```diff
@@ -697,6 +700,8 @@ components:
           type: string
         image_name:
           type: string
+        client_id:
+          type: string
```

Comment:
Can we make this an enum that allows on a select set of values?

Reply:
Most probably yes. It would contain only "ui" for now, but would be expandable. Unlike boolean which I was considering originally. :thinking: :+1: